packages/ndla-ui/src/Article/ArticleByline.tsx

```diff
@@ -45,6 +43,9 @@ const TextWrapper = styled("div", {
     justifyContent: "space-between",
     paddingBlock: "xsmall",
     textStyle: "body.medium",
+    '& [data-contributors="false"]': {
+      marginLeft: "auto",
+    },
```

Comment:
Vi pleier å bruke [logisk](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) styling der det går an. I dette tilfellet kunne du brukt `marginInlineStart`. 

Trenger denne egentlig å wrappes i en condition? Gjør `marginLeft` noe dersom det allerede finnes et element  til venstre for denne blokken?